TitanRobotics2024/src/main/java/frc/robot/Subsystem/Control.java

```diff
@@ -16,6 +19,14 @@ public static Control getInstance()
     public Control()
     {
         driveBase = DriveBase.getInstance();
+        driverController = DriverController.getInstance();
+    }
+
+    public void TeleopControl()
+    {
+        double forward = driverController.getStick(ButtonMap.XboxLEFTSTICKY);
+        double turn = driverController.getStick(ButtonMap.XboxRIGHTSTICKX);
+        driveBase.drive(forward, turn);
     }
```

Comment:
refactor the function name to follow camelCase convention
maybe rename `forward` and `turn` for increased readability